reward/staking_manager.go

```diff
@@ -82,6 +104,9 @@ func (sm *StakingManager) GetStakingInfo(blockNum uint64) *StakingInfo {
 	// Calculate staking info from block header and updates it to cache and db
 	calcStakingInfo, _ := sm.updateStakingInfo(stakingBlockNumber)
 	if calcStakingInfo == nil {
+		if sm.migrationCheck.InMigration() {
```

Comment:
I don't know why this error log is needed?
This case can be happen?

Reply:
Logically, this can never happen. I assume this might happen since we didn't regard stakingInfo DB mandatory.

Reply:
Got it. :)